### ISSUE TITLE
Display bug in Library view

### ISSUE THREAD

**Issue by yihaolee85** [fredrikburmester/streamyfin#73]
**Describe the bug**
Overlapping text in library view. Screenshot should be self explanatory 

**To Reproduce**
1. Go to Library

**Screenshots**
![IMG_1084](https://github.com/user-attachments/assets/a745e176-85a8-4a72-9169-c1c7ef595acc)

**Smartphone (please complete the following information):**
 - Device: iPhone XR
 - OS: iOS 17.6.1
 - Version 0.6.0 (4)

**Comment by fredrikburmester** [comment]
Fixed in a newer version

**Comment by yihaolee85** [comment]
> Fixed in a newer version

@fredrikburmester I downloaded 0.6.2 (3) and I'm still seeing the same, not sure if this was fixed in 0.6.2 (3) or in a future release?

I realized the large "text" was actually embeded as part of the image (this can be seen on the Jellyfin web UI).

If the solution is to remove the smaller text (unless you plan to change the downloaded image), I assume it is at the code here?
https://github.com/fredrikburmester/streamyfin/blob/e911f99b26e1c49763cb0f9792e2927874eeeafb/app/(auth)/(tabs)/libraries/index.tsx#L98-L100
